packages/visual-editor/src/components/pageSections/ReviewsSection.tsx

```diff
@@ -446,13 +483,32 @@ const PageScroller: React.FC<PageScrollerProps> = ({
 const ShowMoreButton: React.FC<{
   expanded: boolean;
   setExpanded: (expanded: boolean) => void;
-}> = ({ expanded, setExpanded }) => {
+  analyticsName: string;
+}> = ({ expanded, setExpanded, analyticsName }) => {
   const { t } = useTranslation();
+  const analytics = useAnalytics();
   return (
     <Button
       className="font-body-fontFamily text-body-fontSize underline cursor-pointer inline-flex items-center gap-2"
-      onClick={() => setExpanded(!expanded)}
+      onClick={() => {
+        expanded // the existing state before toggling
+          ? analytics?.track({
+              action: "EXPAND",
+              eventName: `${analyticsName}-showMore`,
+            })
+          : analytics?.track({
+              action: "COLLAPSE",
+              eventName: `${analyticsName}-showMore`,
+            });
+        setExpanded(!expanded);
+      }}
       variant={"link"}
+      data-ya-action={
+        analytics?.getDebugEnabled() ? "EXPAND/COLLAPSE" : undefined
+      }
+      data-ya-eventname={
+        analytics?.getDebugEnabled() ? `${analyticsName}-showMore` : undefined
```

Comment:
Should this be dependent on the current expanded state to be showMore vs showLess?